core/src/Cuda/Kokkos_Cuda_GraphNodeKernel.hpp

```diff
@@ -61,26 +62,21 @@ class GraphNodeKernelImpl<Kokkos::Cuda, PolicyType, Functor, PatternTag,
   //      attached to the policy?
   // TODO @graph kernel name info propagation
   template <class PolicyDeduced, class... ArgsDeduced>
-  GraphNodeKernelImpl(std::string, Kokkos::Cuda const&, Functor arg_functor,
+  GraphNodeKernelImpl(std::string label_, Cuda const&, Functor arg_functor,
```

Comment:
Dropping the `Kokkos::` qualification introduces an inconsistency with the next constructor...
I don't care too much but it would have been better not to mess with it.

Reply:
- added `label_` and `clang-tidy` complained that I was exceeding the 80 characters
- removed useless `Kokkos::` and made it happy again

We agreed it could stay like that. Closing.